### DISCUSSION TITLE
Where/how should `--always-new-process` be specified?

### DISCUSSION THREAD

**Discussion by lybech** [wezterm/wezterm#2589] [Q&A] [answered]
Hi all, I have a (probably very simple) question:
When I execute `wezterm` from another instance, I get the following info message:
```
wezterm_gui > Spawned your command via the existing GUI instance. Use --always-new-process if you do not want this behavior. Result=SpawnResponse { tab_id: 2, pane_id: 2, window_id: 2, size: TerminalSize { rows: 24, cols: 80, pixel_width: 640, pixel_height: 384, dpi: 0 } }
```
However, if I then try to execute `wezterm --always-new-process`, I get the following error message:
```
error: Found argument '--always-new-process' which wasn't expected, or isn't valid in this context

        If you tried to supply `--always-new-process` as a value rather than a flag, use `-- --always-new-process`
```
Am I misunderstanding something here, or shouldn't this flag be given to `wezterm` on the command line?

**Comment by wez** [comment, marked as answer]
You want `wezterm start --always-new-process`. I've updated that message in 0e98b23392efde37360264fd480402f659afae32
